plugins/machina-tools.js

```diff
@@ -359,13 +359,13 @@ function createInfoLink(text, title, clickCallback) {
   return aside;
 }
 
-machinaTools.onPortalDetailsUpdated = function () {
+machinaTools.onPortalDetailsUpdated = function (data) {
   var portalData;
 
   // If the portal was cleared then exit.
   if (window.selectedPortal === null) return;
 
-  portalData = window.portalDetail.get(window.selectedPortal);
+  portalData = data.portalData;
```

Comment:
Is this change about code improvement (decoupling plugins from cache) rather than fixing a bug?

Reply:
both, this code triggers the issue but it could have used the hook data instead

Reply:
I think I've found a solution to make it backwards compatible. I'll create a separate PR for this